code/modules/mob/living/carbon/species.dm

```diff
@@ -306,7 +303,7 @@ var/global/list/whitelisted_species = list("Human")
 
 		if(H.dna.mutantrace != "slime")
 			switch(breath.temperature)
-				if(-INFINITY to cold_level_3)
+/*				if(-INFINITY to cold_level_3)
```

Comment:
This isn't going to make any sense to anyone who sees it in the future, just remove it and put a comment about it